### PR TITLE
Topology: Add support for MAX98390 echo reference capture

### PR DESCRIPTION
**Pull request by macchian**
codec chip playback loopback to capture stream.

Signed-off-by: Mac Chiang <mac.chiang@intel.com>

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]
@macchian I'm assuming this is validated by you ?

**Comment by lgirdwood** [comment]
@ranj063 fyi.

File: tools/topology/CMakeLists.txt [hunk 1]
```diff
@@ -73,6 +73,7 @@ set(TPLGS
 	"sof-apl-da7219\;sof-apl-da7219"
 	"sof-glk-da7219-kwd\;sof-glk-da7219-kwd\;-DPLATFORM=glk"
 	"sof-glk-da7219-kwd\;sof-cml-da7219-max98357a\;-DPLATFORM=cml"
+	"sof-glk-da7219-kwd\;sof-cml-da7219-max98390\;-DCODEC=MAX98390\;-DPLATFORM=cml"
```
Comment (plbossart) [inline tools/topology/CMakeLists.txt:76]: this file is not known to the kernel, is there any plan to upstream the matching changes for Linux?

**Comment by macchian** [comment]
> @macchian I'm assuming this is validated by you ?

@lgirdwood, it's from Maxim and verified pass on customer board.

File: tools/topology/CMakeLists.txt (hunk 1 above)
Comment (macchian) [reply tools/topology/CMakeLists.txt:76]: @plbossart Thanks for notifying! I upstream the code changes for your reviewing. https://patchwork.kernel.org/project/alsa-devel/patch/1616757689-22014-1-git-send-email-mac.chiang@intel.com/

**Comment by plbossart** [comment]
@macchian is this saying that the MAX98390 uses the same configuration for playback as MAX98357a, but adds an echo reference capture stream? I really want to make sure there's compatibility between the two chips, at least for the SSP playback case.

**Comment by macchian** [comment]
> @macchian is this saying that the MAX98390 uses the same configuration for playback as MAX98357a, but adds an echo reference capture stream? I really want to make sure there's compatibility between the two chips, at least for the SSP playback case.

@plbossart , Yes, max98390 uses the same configuration on playback as max98357a except adding echo reference capture.

**Comment by macchian** [comment]
@plbossart , since master was deleted, should I submit another PR to main branch?

**Comment by paulstelian97** [comment]
@macchian I suggest you do if this is still applicable.

**Pull request closed without merging**